sqlglot/parser.py

```diff
@@ -582,12 +582,37 @@ class Parser(metaclass=_Parser):
     }
 
     CONSTRAINT_PARSERS = {
-        TokenType.CHECK: lambda self: self.expression(
```

Comment:
can you delete all t he tokens? that are no longer used

Reply:
got rid of some of them